### DISCUSSION TITLE
Stats do not detect all zigbee vendors properly

### DISCUSSION THREAD

**Discussion by hpuac** [Koenkk/zigbee2mqtt#22203] [Devices]
Hey folks, when I'm opening the device stats in Z2M I can see that some device vendors are not properly listed:

<img width="276" alt="Screenshot 2024-04-13 at 13 36 03" src="https://github.com/Koenkk/zigbee2mqtt/assets/1850100/eda5e767-5221-46fe-9095-1b596b19c418">

I quickly checked which devices that are:
1. `_TZ3000_ksw8qtmt` = [Nous A1Z](https://www.zigbee2mqtt.io/devices/A1Z.html)
2. `_TZ3000_2putqrmw` = also [Nous A1Z](https://www.zigbee2mqtt.io/devices/A1Z.html)
3. `_TZ3210_r0xgkft5` = [Lidl 14156506L](https://www.zigbee2mqtt.io/devices/14156506L.html)

They work just fine, just don't properly appear in the stats.
Is there any way I can contribute to make them being listed properly? 🙂

**Comment by Koenkk** [comment]
Please create issues related to the frontend in this repository: https://github.com/nurikk/zigbee2mqtt-frontend

**Reply by hpuac** [reply]
Thx, raised it here: https://github.com/nurikk/zigbee2mqtt-frontend/issues/1984
